dev-lang/nasm/nasm-2.14.03rc2.recipe

```diff
@@ -14,24 +14,32 @@ SOURCE_URI="https://www.nasm.us/pub/nasm/releasebuilds/$portVersion/nasm-$portVe
 CHECKSUM_SHA256="1e842d66d250ca02ef91d50f22b239af85b3cd6afb233a8bfbb4502418c13961"
 PATCHES="nasm-$portVersion.patchset"
 
-ARCHITECTURES="x86_gcc2 x86 x86_64"
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
```

Comment:
Please remove this recipe, stay with the stable release, not the release candidate